pkg/execution/state/redis_state/queue.go

```diff
@@ -2208,27 +2235,48 @@ func (q *queue) partitionPeek(ctx context.Context, partitionKey string, sequenti
 		return nil, fmt.Errorf("unknown return type from partitionPeek: %T", peekRet)
 	}
 
-	var encoded, missingPartitions []any
+	var potentiallyMissingPartitions, allPartitionIds []any
 	var totalItemCount int64 // number of items stored in partition ZSET
 	if len(returnedSet) == 3 {
 		totalItemCount, ok = returnedSet[0].(int64)
 		if !ok {
 			return nil, fmt.Errorf("unexpected first item in set returned from partitionPeek: %T", peekRet)
 		}
 
-		encoded, ok = returnedSet[1].([]any)
+		potentiallyMissingPartitions, ok = returnedSet[1].([]any)
 		if !ok {
 			return nil, fmt.Errorf("unexpected second item in set returned from partitionPeek: %T", peekRet)
 		}
 
-		missingPartitions, ok = returnedSet[2].([]any)
+		allPartitionIds, ok = returnedSet[2].([]any)
 		if !ok {
 			return nil, fmt.Errorf("unexpected third item in set returned from partitionPeek: %T", peekRet)
 		}
 	} else if len(returnedSet) != 0 {
 		return nil, fmt.Errorf("expected zero or three items in set returned by partitionPeek: %v", returnedSet)
 	}
 
+	encoded := make([]any, 0)
```

Comment:
This is the same improvement as [above](https://github.com/inngest/inngest/pull/1842/files#r1790611790), just for partition peeks.